torchserve_sanity.sh

```diff
@@ -56,8 +56,8 @@ start_torchserve
 models=("fastrcnn" "fcn_resnet_101" "my_text_classifier_v2" "resnet-18" "my_text_classifier_scripted_v2" "alexnet_scripted" "fcn_resnet_101_scripted"
            "roberta_qa_no_torchscript" "bert_token_classification_no_torchscript" "bert_seqc_without_torchscript")
 
-model_inputs=("examples/object_detector/persons.jpg,docs/images/blank_image.jpg" "examples/image_segmenter/fcn/persons.jpg" "examples/text_classification/sample_text.txt" "examples/image_classifier/kitten.jpg"
- "examples/text_classification/sample_text.txt" "examples/image_classifier/kitten.jpg" "examples/image_segmenter/fcn/persons.jpg" "examples/Huggingface_Transformers/QA_artifacts/sample_text.txt"
+model_inputs=("examples/object_detector/persons.jpg,docs/images/blank_image.jpg" "examples/image_segmenter/persons.jpg" "examples/text_classification/sample_text.txt" "examples/image_classifier/kitten.jpg"
```

Comment:
If the previously mentioned models are included, note that all the sanity check fields should be completed with the information related to those models such as the sample inputs and the required handlers.

Reply:
Postponed for the next iteration.